.github/workflows/build_nuget.yml

```diff
@@ -0,0 +1,110 @@
+name: Windows Nuget
+
+on:
+  push:
+    branches:
+      - master
+      - 'releases/**'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
+  build_nuget_windows:
+    name: nuget.${{ matrix.toolset }}.${{ matrix.arch_config.arch }}
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        os: ["windows-2019"]
+        toolset: ["v141", "v142"]
+        arch_config:
+        - { arch: x64, generator_arch: x64}
+  
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: 'recursive'
+      - uses: ilammy/msvc-dev-cmd@v1
+      - name: Dependencies
+        run: >
+          vcpkg install
+          --host-triplet=${{ matrix.arch_config.arch }}-windows
+          --triplet=${{ matrix.arch_config.arch }}-windows
+          openssl-windows
+          flatbuffers
+          boost-filesystem
+          boost-thread
+          boost-program-options
+          boost-test
+          boost-align
+          boost-math
+          boost-uuid
+          boost-system
+          boost-date-time
+          boost-regex
+          boost-asio
+          boost-interprocess
+      - name: Configure
+        run: >
+          cmake -S . -B build
+          -G "Visual Studio 16 2019" -A ${{ matrix.arch_config.generator_arch }}
+          -T ${{ matrix.toolset }}
+          -DRL_BUILD_NUGET=On
+          -DRL_NUGET_PACKAGE_NAME=RLClientLibNativeStatic
+          -DRL_NUGET_PACKAGE_VERSION="0.1.0"
+          -DNATIVE_NUGET_PLATFORM_TAG=${{ matrix.arch_config.arch }}
+          -DCMAKE_TOOLCHAIN_FILE=C:/vcpkg/scripts/buildsystems/vcpkg.cmake
```

Comment:
It may be better here specify and use the latest version of vcpkg and use VCPKG_ROOT and VCPKG_REF (similar to some vw build processes)

Reply:
According to the documentation, a very recent version of vcpkg is provided on the virtual machine and the path to it is `VCPKG_INSTALLATION_ROOT`.

https://github.com/actions/virtual-environments/blob/main/images/win/Windows2019-Readme.md

I'll add a commit changing the path to use `VCPKG_INSTALLATION_ROOT`